### DISCUSSION TITLE
ccf-validation-tools/Prostate/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#277] [General]
# ccf-validation-tools/Prostate/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Prostate/comments/

**Comment by emquardokus** [comment]
Prostate_v1.1_DRAFT
https://docs.google.com/spreadsheets/d/1hlSptGNXzyM7vxsH930YMf6gZkHVgHUE-Qc_4uFAmoU/edit#gid=1239199370

**Blank ontology ID missing parent**
ADDED CT/1 as basal cell of urothelium CL:1000486    In row 25, the term basal cell of prostatic urethra without ontology ID has no parent that is from the CL ontology.

**Comment by emquardokus** [comment]
Prostate_v1.1_DRAFT
https://docs.google.com/spreadsheets/d/1hlSptGNXzyM7vxsH930YMf6gZkHVgHUE-Qc_4uFAmoU/edit#gid=1239199370

Some substantial changes were made by author Doug Strand, need to inquire intent on the blood vasculature work, this new area seems problematic; seems like there was a copy paste column shift too.


**Different labels**

1. ADDED: Label was vascular system, added blood vasculature	UBERON:0004537 instead of the uberon in this issue    In row 19, the term UBERON:0001085 has different name/label in the source ontology. The name/label in the ASCT+B table is vascular system and the one in the ontology is skin of trunk. For reference, the given name/label by SMEs is vascular system. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

1.  Removed because misalgined   In row 24, the term UBERON:0002441 has different name/label in the source ontology. The name/label in the ASCT+B table is pericyte and the one in the ontology is cervicothoracic ganglion. For reference, the given name/label by SMEs is pericyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

2. ADDED: body proper	UBERON:0013702 since this is what we use in the HRA for body     In row 19, the term UBERON:0001489 has different name/label in the source ontology. The name/label in the ASCT+B table is whole body and the one in the ontology is manus joint. For reference, the given name/label by SMEs is whole body. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

3. ADDED  blood vessel smooth muscle   In row 23, the term UBERON:0001431 has different name/label in the source ontology. The name/label in the ASCT+B table is vascular smooth muscle and the one in the ontology is distal carpal bone 2. For reference, the given name/label by SMEs is vascular smooth muscle. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

4. ADDED correct uberon ID rows 19, 20, 21, 23, 24 blood vessel UBERON:0001981   In row 19, the term UBERON:0001102 has different name/label in the source ontology. The name/label in the ASCT+B table is blood vessel and the one in the ontology is cartilage of main bronchus. For reference, the given name/label by SMEs is blood vessel. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

5.  ADDED correct lymphatic vessel	UBERON:0001473 In row 22, the term UBERON:0004167 has different name/label in the source ontology. The name/label in the ASCT+B table is lymphatic endothelial cell and the one in the ontology is orbitofrontal cortex. For reference, the given name/label by SMEs is lymphatic endothelial cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

6.  ADDED club-like cell of the urethral epithelium   In row 26, the term CL:4032000 has different name/label in the source ontology. The name/label in the ASCT+B table is club cell of prostatic urethral epithelium and the one in the ontology is club-like cell of the urethral epithelium. For reference, the given name/label by SMEs is club epithelia. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Blank ontology ID**

1. ADDED ionocyte	CL:0005006    In row 15, no term id was found for the name/label ionocyte epithelia.

2. Misaligned columns; fixed    In row 19, no term id was found for the name/label endothelial cell of artery.

3. Misaligned columns; fixed     In row 20, no term id was found for the name/label capillary endothelial cell.

4. Misaligned columns; fixed     In row 21, no term id was found for the name/label vein endothelial cell.

4. Misaligned columns; fixed     In row 22, no term id was found for the name/label lymph node lymphatic vessel endothelial cell.

**Comment by emquardokus** [comment]
Prostate_v1.1
https://docs.google.com/spreadsheets/d/1hlSptGNXzyM7vxsH930YMf6gZkHVgHUE-Qc_4uFAmoU/edit#gid=1239199370

**Terms not found**
Corrected: The term used in table was: hematopoietic system	UBERON:000570, which should be UBERON:0002390

===
Corrected: The term used in table was blood vessel endothelium UBERON:0000766 , which should be UBERON:0004638

**Typos or punctuation mistakes**
FIXED:
    It might have a typo in the term NEED in the following 2 rows 17, 18. The term id should have this pattern: UBERON:NNNNNNN or CL:NNNNNNN or PCL:NNNNNNN. The ontology name in upper case. N is a number, and it should have exactly 7 numbers after the colon. Please change it in the ASCT+B table.

**Different labels**

1.  CORRECTED: The term CL:0000413 has a different name/label in the source ontology in the following 1 row 19. The name/label in the ASCT+B table is endothelial cell of artery and the one in the ontology is haploid cell. For reference, the given name/label by SMEs is endothelial cell of artery. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

The intended cell type was CL:1000413 endothelial cell of artery; note one number off in CL ID between them CL:0000413 haploid cell.

2. CORRECTED 2 instances: The term UBERON:0001085 has a different name/label in the source ontology in the following 1 row 23. The name/label in the ASCT+B table is vascular system and the one in the ontology is skin of trunk. For reference, the given name/label by SMEs is vascular system. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

The intended term was vascular system	UBERON:0007798

3. CORRECTED   removed cell type from AS column; correct CT was already in CT column. The term UBERON:0004167 has a different name/label in the source ontology in the following 1 row 22. The name/label in the ASCT+B table is lymphatic endothelial cell and the one in the ontology is orbitofrontal cortex. For reference, the given name/label by SMEs is lymphatic endothelial cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Prostate_v1.1
https://docs.google.com/spreadsheets/d/1hlSptGNXzyM7vxsH930YMf6gZkHVgHUE-Qc_4uFAmoU/edit#gid=1239199370
The term The term used in table was: hematopoietic system was replaced with lymph vasculature	lymph vasculature	UBERON:0004536 as parent of lymphatic vessel	UBERON:0001473
